ci_framework/roles/artifacts/tasks/edpm.yml

```diff
@@ -77,8 +77,7 @@
           sudo test -d /var/lib/openstack && sudo cp -a /var/lib/openstack /tmp/{{ host_ip }}
           sudo test -d /var/lib/config-data && sudo cp -a /var/lib/config-data /tmp/{{ host_ip }}
           sudo test -d /var/lib/cloud && sudo cp -a /var/lib/cloud /tmp/{{ host_ip }}
-          sudo test -d /etc/nftables && sudo cp -a /etc/nftables /tmp/{{ host_ip }}
-          sudo test -d /etc/os-net-config && sudo cp -a /etc/os-net-config /tmp/{{ host_ip }}
+          sudo test -d /etc && sudo cp -RP /etc /tmp/{{ host_ip }}
```

Comment:
It's very far fetched. Can you be more specific?
There is another PR https://github.com/openstack-k8s-operators/ci-framework/pull/575 which is planning to use `sosreport`.
It would be beneficial to know what the files are you looking for.

Reply:
As I wrote above for @cjeanner:
> I know what I need today, the /etc/ssh/ssh_know_hosts file, but I cannot predict what I will need in my next CI troubleshooting. I agree about the recursion issue though. What about cp -RP /etc ? That would follow none of the symlink

Reply:
Having a full sos report form the EDPM nodes would be nice